cli/src/main/java/org/jboss/gm/cli/Main.java

```diff
@@ -111,8 +119,62 @@ public int run(String[] args) throws Exception {
         return result;
     }
 
-    private void executeGradle() throws ManipulationException {
+    private void logAndThrow(String message, Throwable cause) throws ManipulationException {
```

Comment:
If we a simplifying as per the other comments I don't think we need this indirection

Reply:
I've removed this method.